rasgotransforms/rasgotransforms/transforms/funnel/funnel.sql

```diff
@@ -1,7 +1,7 @@
 {%- for i in range((hierarchy|length) - 1) -%}
     SELECT
-    {{ hierarchy[i] }} AS SOURCE_NODE,
-    {{ hierarchy[i+1] }} AS DEST_NODE,
+    CAST({{ hierarchy[i] }} AS STRING) AS SOURCE_NODE,
```

Comment:
Do we need to 0 pad the strings so we don't get sorting = 1, 11, 12, ..., 2?

Or will the UI handle that for us?

Reply:
the order of `source_node` and `dest_node` doesn't matter - the vis code in the UI matches sources and destinations